web/src/popover_menus.js

```diff
@@ -967,6 +967,11 @@ export function initialize() {
                 });
             });
 
+            $(".focus-dropdown").on("focus", (e) => {
+                e.preventDefault();
+                $popper.find("li:not(.divider):visible a").eq(0).trigger("focus");
```

Comment:
I don't see `.divider` class being used.

Reply:
You can add this back based on my comment above https://github.com/zulip/zulip/pull/25667#discussion_r1266453935